owncloudApp/src/main/java/com/owncloud/android/dependecyinjection/LocalDataSourceModule.kt

```diff
@@ -63,16 +66,16 @@ val localDataSourceModule = module {
     single { OwncloudDatabase.getDatabase(androidContext()).transferDao() }
     single { OwncloudDatabase.getDatabase(androidContext()).spacesDao() }
 
-    single<SharedPreferencesProvider> { OCSharedPreferencesProvider(get()) }
+    singleOf(::OCSharedPreferencesProvider) bind SharedPreferencesProvider::class
     single<LocalStorageProvider> { ScopedStorageProvider(dataFolder, androidContext()) }
 
     factory<LocalAppRegistryDataSource> { OCLocalAppRegistryDataSource(get()) }
```

Comment:
Can't this be converted to the new syntax as well? We just have a `get()`, or am I missing something?